route/test/test_route_generator.cpp

```diff
@@ -47,7 +47,7 @@ Using this file:
 */
 
 
-TEST(RouteGeneratorTest, testRouteVisualizerCenterLineParser)
+TEST(RouteGeneratorTest, DISABLED_testRouteVisualizerCenterLineParser)
```

Comment:
Is this `DISABLED` prefix doing something, or just a note for us? If the tests aren't needed any more, I'd say we should delete or replace them, if they are needed but are disabled because they're broken, we need to fix them.

Reply:
That prefix is in reference to [this issue](https://github.com/usdot-fhwa-stol/carma-platform/issues/1013) that I noticed when I started unit testing. I'm not sure if it has been addressed yet.